### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by tursynbekoff**
First PR

created boilerplate nextjs project

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  return (
+    <>
+      <Head>
+        <title>Create Next App</title>
+        <meta name="description" content="Generated by create next app" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <main className={styles.main}>
+        <div className={styles.description}>
```
Comment (olzh2102) [inline pages/index.tsx]: remove all these crap between `main` tags

File: styles/Home.module.css [hunk 2]
```diff
@@ -0,0 +1,278 @@
+.main {
```
Comment (olzh2102) [inline styles/Home.module.css]: this file can be removed as well and its related `import`

**Review by olzh2102 — CHANGES_REQUESTED** [review]

**Comment by olzh2102** [comment]
inside `/public` folder, all the `.svg` files can be removed as well

**Review by olzh2102 — APPROVED** [review]

**Comment by tursynbekoff** [comment]
LGTM desh

**Pull request merged**